packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -315,6 +315,8 @@ class Auth {
   void _loadCredentials() {
     final envToken = platform.environment[shorebirdTokenEnvVar];
     if (envToken != null) {
+      logger.info('$shorebirdTokenEnvVar detected');
```

Comment:
shouldn't this be a logger.detail? info is probably too noisy for this type of diagnostic log imo 🤷

Reply:
I originally wrote this as a detail but opted for info because it's just two lines and will only ever print on CI. I don't feel strongly.